packages/labs/analyzer/src/test/analyzer_test.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {test} from 'uvu';
+import * as assert from 'uvu/assert';
+import ts from 'typescript';
+import * as path from 'path';
+
+import {Analyzer} from '../lib/analyzer.js';
+
+test('Basic elements - isLitElement', () => {
+  const basicElementsPackagePath = new URL(
+    '../test-files/basic-elements',
+    import.meta.url
+  ).pathname;
```

Comment:
Speaking of Windows support, I think it would be a good idea to start testing on Windows right away, since it's easier to handle Windows edge cases incrementally instead of in a big pass afterwards. It's pretty to set up Windows CI with GitHub Actions, see https://github.com/lit/wireit/blob/97b7a9f3932597a8c91d3a4d406fcdd7251d234d/.github/workflows/tests.yml#L9

(Another PR fine, I just recommend doing it sooner rather than later)